src/services.py

```diff
@@ -1,107 +0,0 @@
-import logging
-import os
-
-import pandas as pd
-
-import src.constants as const
-from src.database import SQLiteDatabase
-from src.sql_templates import (
-    get_bonds_recon_report_with_fund_name_and_reporting_date_query,
-    get_distinct_fund_name_and_reporting_date_query,
-    get_distinct_reporting_date_query,
-    get_equity_recon_report_with_fund_name_and_reporting_date_query,
-    get_rate_of_return_query,
-    get_recon_report_query,
-)
-from src.utils import get_best_performing_funds
-
-
-def generate_recon_report_for_equity(db: SQLiteDatabase) -> pd.DataFrame:
-    fund_name_and_reporting_date = db.fetch_as_df(get_distinct_fund_name_and_reporting_date_query())
-
-    recon_df = pd.DataFrame()
-
-    for _, row in fund_name_and_reporting_date.iterrows():
-        fund_name, reporting_date = row['FUND NAME'], row['REPORTING DATE']
-        logging.info(f"Running EQUITY recon report for fund name: {fund_name}, reporting date: {reporting_date}")
-
-        recon_report_equity_query = get_equity_recon_report_with_fund_name_and_reporting_date_query(fund_name,
-                                                                                                      reporting_date)
-        recon_equity_results = db.fetch_as_df(recon_report_equity_query)
-
-        if not recon_equity_results.empty:
-            logging.warning(f"Require EQUITY reconciliation for fund name: {fund_name}, reporting date: {reporting_date}!")
-            recon_df = pd.concat([recon_df, recon_equity_results], ignore_index=True)
-
-    logging.info("Completed recon report for EQUITY")
-
-    return recon_df.reset_index(drop=True)
-
-def generate_recon_report_for_bonds(db: SQLiteDatabase) -> pd.DataFrame:
```

Comment:
Note that I mentioned in the workshop that only equities needed to be considered...